### ISSUE TITLE
run_allometry error: ValueError('No parameters found')

### ISSUE THREAD

**Issue by liuxl17** [pharmpy/pharmpy#1044]
This error occurred when testing the `run_amd` with warfarin dataset. Below is the detailed error information:
```
Key:       add allometry-run0-results
Function:  execute_task
args:      ((<function results at 0x2abccf7f6160>, <Pharmpy model object best_resmod_1>, (<function retrieve_from_database_or_execute_model_with_tool.<locals>.task at 0x2abd31d38a60>, (functools.partial(<function _add_allometry_on_model at 0x2abccf7f60d0>, allometric_variable='WT', reference_value=70, parameters=None, initials=None, lower_bounds=None, upper_bounds=None, fixed=True), <Pharmpy model object best_resmod_1>))))
kwargs:    {}
Exception: "ValueError('No parameters found')"
```
I then checked the AMD user guide. The description in the "AMD" (https://pharmpy.github.io/latest/amd.html#allometry) is different from those specified in the "allometry" (https://pharmpy.github.io/latest/allometry.html#running). The default setting of parameters to apply scaling to is supposed be all CL, Q and V parameters but it is `None` in this case, so do `initials`, `lower_bounds`, and `upper_bounds`.

There are no detailed arguments in the example code but I tried to write by myself and tested the `run_allometry` subtool individually, using the final model from `resmod` module as an input. Below is the input model and my code:

Input model
```
$PROBLEM [VC,CL]+[MAT,MDT]
$INPUT ID TIME WT AGE SEX AMT DVID DV MDV
$DATA best_resmod_1.csv IGNORE=@  
$SUBROUTINES ADVAN5  TRANS1 
$MODEL COMPARTMENT=(TRANSIT1 DEFDOSE) COMPARTMENT=(TRANSIT2) COMPARTMENT=(TRANSIT3) COMPARTMENT=(TRANSIT4) COMPARTMENT=(TRANSIT5) COMPARTMENT=(TRANSIT6) COMPARTMENT=(TRANSIT7) COMPARTMENT=(TRANSIT8) COMPARTMENT=(TRANSIT9) COMPARTMENT=(TRANSIT10) COMPARTMENT=(DEPOT) COMPARTMENT=(CENTRAL)
$PK
MDT = THETA(4)*EXP(ETA(4))
VC = THETA(1) * EXP(ETA(1))
CL = THETA(2) * EXP(ETA(2))
MAT = THETA(3) * EXP(ETA(3))
KA = 1/MAT
V = VC
K12 = 10/MDT
K23 = 10/MDT
K34 = 10/MDT
K45 = 10/MDT
K56 = 10/MDT
K67 = 10/MDT
K78 = 10/MDT
K89 = 10/MDT
K9T10 = 10/MDT
K10T11 = 10/MDT
K11T12 = KA
K12T0 = CL/V
$ERROR
IF (A(12)/VC.EQ.0) THEN
    IPREDADJ = 2.22500000000000E-16
ELSE
    IPREDADJ = A(12)/VC
END IF
Y = A(12)/VC + A(12)*EPS(1)/VC + EPS(2)
$ESTIMATION METHOD=COND INTERACTION MAXEVAL=9999 SADDLE_RESET=1 
$TABLE ID TIME CWRES CIPREDI VC DV PRED RES WRES IPREDI IWRESI FILE=mytab NOAPPEND NOPRINT
$THETA (0, 8.01621, Inf) ; POP_VC
$THETA (0, 0.131785, Inf) ; POP_CL
$THETA (0, 0.445779, Inf) ; POP_MAT
$THETA  (0,0.987272) ; POP_MDT
$OMEGA BLOCK(2)
0.0479822	; IIV_VC
0.0258434	; IIV_VC_IIV_CL
0.0783349	; IIV_CL
$OMEGA BLOCK(2)
1.89185	; IIV_MAT
-0.676248	; IIV_MAT_IIV_MDT
0.391653	; IIV_MDT
$SIGMA  0.702575 ; sigma_prop
$SIGMA  2.49449 ; sigma_add
```
Testing code
```
library(pharmr)

model <- read_model('best_resmod_1.mod')
fit(model)

res <- run_allometry(model=model,
                     allometric_variable = 'WT',
                     reference_value = 70,
                     parameters = list('VC', 'CL'),
                     initials = c(1, 0.75))
```
And this code works. So I suppose this error is related to the setting of `parameters` and `initials`. 

I don't know if it is possible to specify these two arguments in the `run_amd` function. Also, I am not sure why this only happened in one single dataset, other datasets also accept the same default setting, which is `None`, but do not have this error.

**Comment by ZheHuang96** [comment]
Do you mean other datasets also have `parameters=None, initials=None, lower_bounds=None, upper_bounds=None` ?

**Comment by liuxl17** [comment]
> Do you mean other datasets also have `parameters=None, initials=None, lower_bounds=None, upper_bounds=None` ?

Yes. It is the default setting.

**Comment by pharmpy-dev-123** [comment]
Will be fixed by #1067.
